src/core/lib/surface/server.h

```diff
@@ -213,6 +217,12 @@ class Server : public ServerInterface,
   void SendGoaways() ABSL_LOCKS_EXCLUDED(mu_global_, mu_call_);
 
  private:
+  // note: the grpc_core::Server redundant namespace qualification is
+  // required for older gcc versions.
+  friend absl::Status(::grpc_server_add_passive_listener)(
```

Comment:
Please add a TODO assigned to @yashykt to eliminate this `friend` statement as part of his upcoming server listener refactoring.

Reply:
Done